scanners/boostsecurityio/osv-scanner/module.yaml

```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/osv-scanner
-name: BoostSecurity osv-scanner
+name: OSV Scanner
```

Comment:
According to https://github.com/google/osv-scanner maybe it should be OSV-Scanner  with a dash